linkcheck/checker/mailtourl.py

```diff
@@ -145,57 +152,110 @@ def check_email_syntax(self, mail):
         # restrict email length to 256 characters
         # http://www.rfc-editor.org/errata_search.php?eid=1003
         if len(mail) > 256:
-            self.set_result(_("Mail address `%(addr)s' too long. Allowed 256 chars, was %(length)d chars.") % \
-            {"addr": mail, "length": len(mail)}, valid=False, overwrite=False)
+            self.set_result(
+                _(
+                    "Mail address `%(addr)s' too long. Allowed 256 chars, "
+                    "was %(length)d chars."
```

Comment:
I mean, if all existing code uses the Objectively Incorrect wrapping style with spaces at the ends of lines, then I guess I could be convinced that consistency trumps Correctness :tm:.

Reply:
Fixed.